lib/ftp-remote-edit.js

```diff
@@ -1199,7 +1198,8 @@ class FtpRemoteEdit {
     }
 
     let initialPath, initialName, initialType, ref;
```

Comment:
Use let instead of var

let entry, initialPath, initialName, initialType, ref;